docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -485,62 +485,87 @@ More information on the methods can be found in the `demographics methods append
 4.0 Results
 +++++++++++
 
-As mentioned above, before beginning our research, we should determine the variables
-of interest: the facets of gender and/or sex that we expect to be most directly associated with
-our outcome.
-If it isn't possible to narrow this down or distinguish what might be important, we can consider
-all gender- and sex-related factors to be variables of interest.
+4.1 Goals when Reporting Results
+--------------------------------
 
-When we present our findings, we have two goals:
+When we present our findings, we have two primary goals:
 
-* Accurately convey what associations we are looking for, and what the causal structures might be.
+* Accurately convey the variables of interest, and what the causal structures might be.
   We should strive to be as precise as possible, and not rely on "common sense"
   to indicate which gender- and sex-related factors are related to a health outcome.
   Despite prevailing belief that sex affects gender but not vice versa, there is
   strong evidence for causal influence in both directions. [Ritz_and_Greaves_2022]_ [Springer_2012]_
-  Even when we do think that a sex-related factor is most proximal to our outcome,
+  Even when we do think that a sex-related factor is most causally related to our outcome,
   we should still mention that some of its impact could be mediated through
   gender factors and that it is difficult to disentangle these.
 * Highlight data limitations around sex and gender measures.
   We don't want our inclusive language to give the impression that we are able to make accurate
-  conclusions about sexual and gender minority populations when the data prevents us from doing this.
+  conclusions about sexual and gender minority populations when the data prevent us from doing this.
   Furthermore, drawing attention to the problems with how almost every data source currently
```

Comment:
I know this isn't new, but just wanted to emphasize how important this point is!! The Rioux et al (2022) paper also made a similar point, that inclusive language/reporting can work in tandem (and become less difficult/complicated) with improved data collection methods. 

_Accordingly, since the gender of pregnant participants is currently rarely reported or measured, our recommendations favour gender-inclusive language as more accurate language. In the future however, the field should move towards measuring and reporting participants’ gender modality/identity as any other demographic measures. As this becomes the norm, cis women-related terms and language inclusive of gender diversity will be able to better coexist by qualifying sampled participants and studied pregnant populations precisely and appropriately, which will avoid erasure of both cisgender women and gender-diverse pregnant people._

I wonder if we should add something similar to this in our Guiding Principles, like the 'This is Work' section.

Reply:
It would be great to cite this Rioux et al paper! I think it could support several different parts of the guide 😃